cornflow/endpoints/permission.py

```diff
@@ -40,8 +46,29 @@ def get(self):
         """
         return PermissionViewRoleModel.get_all_objects()
 
-    def post(self):
-        return EndpointNotImplemented()
+    @doc(description="Create a new permission", tags=["PermissionViewRole"])
+    @Auth.auth_required
+    @use_kwargs(PermissionViewRoleRequest, location="json")
+    @marshal_with(PermissionViewRoleResponse)
+    def post(self, **kwargs):
+        AUTH_TYPE = current_app.config["AUTH_TYPE"]
+        if AUTH_TYPE == AUTH_LDAP:
+            raise EndpointNotImplemented(
+                "The permissions have to be created in the directory."
+            )
+
```

Comment:
Actually, if I remember this correctly, but I should review it, the permissions are not defined in the directory, just the roles, but the permission assigned to each role should be done in cornflow. Do not fix it just yet until I review it.